src/components/pages/Version.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {RouteComponent} from "../../features/main/Route";
+import {BodyLong,  Box, Heading,  VStack} from "@navikt/ds-react";
+import SupportTemplate from "../templates/SupportTemplate";
+import {Contact} from "../atoms/Contact";
+import {ABOUT_VERSIONS, NOVEMBER23} from "../utils/version/VersionTexts";
+
+const Version: RouteComponent = () => {
+
+    return (
+        <SupportTemplate keyPrefix={'pages.versjon'}>
```

Comment:
Liker versjoneringen, enkelt å oversiktlig. 
imo trenger det ikke være to panel her, det kan heller være samlet i en med info på toppen og lista under? Tanker?
<img width="1490" alt="image" src="https://github.com/FINTLabs/fint-flyt-integration-frontend/assets/10653250/7a237810-9a0b-414f-9cc0-ea1071e29b63">

Reply:
ja why not ? 
vi lager en listevisning av det